src/data/faq_de.json

```diff
@@ -550,33 +550,33 @@
                                 ]
                             },
                             {
                                 "title": "Wie lange bleibt der Test in der App zu sehen?",
                                 "anchor": "warn_without_tan_in_app_display",
                                 "active": false,
                                 "textblock": [
-                                    "Die Warnung anderer mit der neuen Funktion \"Tests erfassen und andere warnen\" führt gar nicht zur Anzeige eines positiven Testergebnisses in der Corona-Warn-App.",
-                                    "Für Tests, die wie bisher in der App registriert werden, gilt nach wie vor: Tests stellen in der Diagnostik von Infektionskrankheiten immer nur eine Momentaufnahme dar, sie sind daher nicht unbegrenzt lange gültig. Nach 72 Stunden wird daher ein registrierter Test in der Corona-Warn-App als „veraltet“ grau dargestellt."
+                                    "Die Warnung anderer mit der neuen Funktion \"Tests erfassen und andere warnen\" führt gar nicht zur Anzeige eines positiven Testergebnisses in der Corona-Warn-App, sondern dient lediglich dazu, Ihre Kontakte zu warnen.",
+                                    "Für Tests, die wie bisher in der App registriert werden, gilt nach wie vor: Tests stellen in der Diagnostik von Infektionskrankheiten immer nur eine Momentaufnahme dar, sie sind daher nicht unbegrenzt lange gültig. Nach 72 Stunden wird daher ein registrierter Test in der Corona-Warn-App als „veraltet“ angezeigt."
                                 ]
                             },
                             {
-                                "title": "Stoppt nach dem Teilen des positiven Ergebnisses meine Risikoermittlung? Wie lange?",
+                                "title": "Stoppt nach dem Teilen des positiven Ergebnisses meine Risiko-Ermittlung?",
                                 "anchor": "warn_without_tan_risk_assessment",
                                 "active": false,
                                 "textblock": [
-                                    "Nein, die Risikoermittlung und das Contact Tracing Protokoll werden nicht gestoppt. Die Warnung anderer mit den neuen Funktion \"Tests erfassen und andere warnen\" hat keine Auswirkungen auf die Risikoermittlung.",
-                                    "Wenn Sie sich trotz positiven Tests in der Öffentlichkeit bewegen, achten Sie bitte auf die Abstands- und Hygieneregeln und tragen Sie, wenn möglich, eine Maske zur Verringerung einer Infektionsgefahr für andere."
+                                    "Nein, die Risiko-Ermittlung wird nicht gestoppt. Die Corona-Warn-App zeichnet also weiterhin ihre Kontakte auf und sendet Zufalls-IDs aus. Die Warnung anderer mit der Funktion \"Tests erfassen und andere warnen\" hat also keine Auswirkungen auf die Risiko-Ermittlung.",

```

Comment:
"ihre Kontakte" => "Ihre Kontakte"

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/3322/commits/ff61d079ed080cd3b21a3a6996709f3b3f04ad26, thanks!